DMTN-222.tex

```diff
@@ -245,10 +245,19 @@ \subsection{Distribution}
 \label{sec:calib_export}
```

Comment:
The changelog (located somewhere above this line) should be updated as well, as well as the author list if you're not already on it.